locales/en.po

```diff
@@ -1027,8 +1027,8 @@ msgid "Mood"
 msgstr "Mood"
 
 #: [mood]im
-msgid "I'm "
-msgstr "I'm "
+msgid "I’m "
```

Comment:
This will need to be changed in the source, as it will be very hard to translate in languages with a non-English word order.